Riot/Modules/SideMenu/SideMenuCoordinator.swift

```diff
@@ -470,78 +467,13 @@ extension SideMenuCoordinator: SideMenuNavigationControllerDelegate {
 //            case .addSpace:
 //                AppDelegate.theDelegate().showAlert(withTitle: VectorL10n.spacesAddSpace, message: VectorL10n.spacesFeatureNotAvailable(AppInfo.current.displayName))
 //            case .settings:
-//                if #available(iOS 14.0, *) {
-//                    self.showSpaceSettings(spaceId: spaceId, session: session)
-//                } else {
-//                    AppDelegate.theDelegate().showAlert(withTitle: VectorL10n.settingsTitle, message: VectorL10n.spacesFeatureNotAvailable(AppInfo.current.displayName))
-//                }
+//                self.showSpaceSettings(spaceId: spaceId, session: session)
 //            case .invite:
 //                self.showSpaceInvite(spaceId: spaceId, session: session)
 //            }
 //        }
 //    }
 //}
-=======
-extension SideMenuCoordinator: SpaceListCoordinatorDelegate {
```

Comment:
Should we keep this code coming from element and comment it ?

Reply:
The code is still here in the latest commit. Sorry about that.